src/core/parser.zsh

```diff
@@ -103,6 +103,33 @@ _fzf_complete_parse_argument() {
     return $(( index > #command_arguments ))
 }
 
+_fzf_complete_parse_option() {
+    local result=()
+    local shorts=(${(z)1})
+    local longs=(${(z)2})
+    local options_argument_required=(${(z)3})
+    shift 3
+
+    local cmd=(${(Q)${(z)@}})
+    local cmd_shorts=(${(M)cmd:#-[^-]*})
+
+    local option_argument_required
+    for option_argument_required in $options_argument_required; do
+        cmd_shorts=(${cmd_shorts%%${option_argument_required#-}*})
+    done
```

Comment:
If I understood these lines correctly, they look like they are equivalent to:
```zsh
cmd_shorts=${cmd_shorts:|options_argument_required}
```

Reply:
This removes the short options that require an argument and after. Using example https://github.com/chitoku-k/fzf-zsh-completions/pull/130#discussion_r640032299, the behaviour of the comment removes `C` only, but the characters `C` and after are removed in this implementation.

Reply:
Thanks for clarification!